assets/trees/tableschema.json

```diff
@@ -0,0 +1,215 @@
+{
+  "fields": [
+    {
+      "name": "lat",
+      "title": "Latitude in decimal degrees (EPSG:4326)"
```

Comment:
There are errors on this JSON file, do you still want to keep it?

Reply:
Ops! Forgot to remove it.